packages/react/.storybook/preview.js

```diff
@@ -1,13 +1,119 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { white, g10, g90, g100 } from '@carbon/themes';
 import { breakpoints } from '@carbon/layout';
-import { GlobalTheme } from '@carbon/react/es/components/Theme';
+import { GlobalTheme, Theme } from '@carbon/react/es/components/Theme';
 import { Layout } from '@carbon/react/es/components/Layout';
 import { TextDirection } from '@carbon/react/es/components/Text';
+import { DocsContainer, Meta, Unstyled } from '@storybook/blocks';
+import {
+  H1,
+  H2,
+  H3,
+  H4,
+  H5,
+  H6,
+  P,
+  OL,
+  UL,
+  LI,
+  Link,
+  Blockquote,
+  PageTable,
+  Caption,
+  Code,
+  AnchorLinks,
+  AnchorLink,
+  Column,
+  DoDont,
+  DoDontRow,
+  GifPlayer,
+  PageDescription,
+  Row,
+  StorybookDemo,
+  Grid,
+  Tabs,
+  InlineNotification,
+  Tab,
+  ArtDirection,
+} from '../src/components/MDXComponents';
+import '../src/components/MDXComponents/components/index.scss';
```

Comment:
Same with this. Does the `@carbon-labs/mdx-components` package export the index.scss file? Should we do the Lerna workspace import here or keep it as a relative path?

Reply:
same as above, it works natively